src/utils/ModelUtils.js

```diff
@@ -103,8 +103,7 @@ export function getServerRoles (model) {
 // Merges the relevant properties of destination server into the src and returns the merged version.  Neither
 // src or dest are modified
 export function  getMergedServer (src, dest, props)  {
-  let result = Object.assign({}, src);
-
+  let result = Object.assign({}, src instanceof Map ? src.toJS() : src);
```

Comment:
src can be either a map or not a map? that seems a bit odd...

Reply:
Yeah, it is used in multiple places....if it is a map...it put some wired entries.you can try without it and go to validator to see the servers file.......toJS sort of get rid of those wired Map entries.